lib/Sass/common/_variables-export.scss

```diff
@@ -6,13 +6,29 @@
 @import "terriajs-variables";
 
 :export {
+  // TODO: find a better solution for this - some of these we need to provide as
+  // numbers only so we can do math on them and add the unit afterwards 😵
+  // mobile: $mobile;
+  // sm: $sm;
+  // md: $md;
+  // lg: $lg;
+  mobile: 767;
```

Comment:
These are widths, yeah? Which is reasonably self evident but it might be worth sticking a comment in here to make it explicit

Reply:
Tidied them a little 🙂